ts/src/luno.ts

```diff
@@ -145,8 +142,9 @@ export default class luno extends Exchange {
                 'trading': {
                     'tierBased': true, // based on volume from your primary currency (not the same for everyone)
                     'percentage': true,
-                    'taker': this.parseNumber ('0.001'),
-                    'maker': this.parseNumber ('0'),
+                    // Default values differ per counter. Below is for South Africa
```

Comment:
@DelAnt can you point me to the link showing the fees?

Reply:
https://www.luno.com/en/countries/AU
Listed per country.

Reply:
@DelAnt thanks, yeah but in this case I would keep it as is, most our users are from europe/asia so I don't think we should make South Africa de default one

Reply:
I hear that. Worth taking into account is that Luno's trade volume is by and large South Africa. America has no trading exchange, while Europe is only a smallest volume. 
The problem I am trying to solve is that once markets a loaded, these maker/trader fees are copied into each market pair, but are actually invalid, and not tier based. 
Is there any method to update market fee values after calling fetchTradingFee()